.drone.star

```diff
@@ -2880,12 +2897,12 @@ def middlewareService(ocis = False, federatedServer = False):
 
     return [{
         "name": "middleware",
-        "image": "owncloud/owncloud-test-middleware:1.0.0",
+        "image": "owncloud/owncloud-test-middleware",
```

Comment:
```suggestion
        "image": "owncloud/owncloud-test-middleware:1.1.0",
```

Before merging, this should specify the tag that will soon be added to the middleware repo.

Reply:
fixed